simple-manager-blueprint.yaml

```diff
@@ -92,96 +92,96 @@ inputs:
   # agent installer to identify it for your distro and release automatically.
   agent_package_urls:
     default:
-      ubuntu_trusty_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/Ubuntu-trusty-agent_3.4.0-m5-b394.tar.gz
-      ubuntu_precise_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/Ubuntu-precise-agent_3.4.0-m5-b394.tar.gz
-      centos_7x_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/centos-Core-agent_3.4.0-m5-b394.tar.gz
-      centos_6x_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/centos-Final-agent_3.4.0-m5-b394.tar.gz
-      redhat_7x_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/redhat-Maipo-agent_3.4.0-m5-b394.tar.gz
-      redhat_6x_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/redhat-Santiago-agent_3.4.0-m5-b394.tar.gz
-      cloudify_windows_agent: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/cloudify-windows-agent_3.4.0-m5-b394.exe
+      ubuntu_trusty_agent: Ubuntu-trusty-agent_3.4.0-m5-b394.tar.gz
+      ubuntu_precise_agent: Ubuntu-precise-agent_3.4.0-m5-b394.tar.gz
+      centos_7x_agent: centos-Core-agent_3.4.0-m5-b394.tar.gz
+      centos_6x_agent: centos-Final-agent_3.4.0-m5-b394.tar.gz
+      redhat_7x_agent: redhat-Maipo-agent_3.4.0-m5-b394.tar.gz
+      redhat_6x_agent: redhat-Santiago-agent_3.4.0-m5-b394.tar.gz
+      cloudify_windows_agent: cloudify-windows-agent_3.4.0-m5-b394.exe
 
   #############################
   # Cloudify Modules
   #############################
   rest_service_rpm_source_url:
     type: string
-    default: http://repository.cloudifysource.org/org/cloudify3/3.4.0/m5-RELEASE/cloudify-rest-service-3.4.0-m5_b394.x86_64.rpm
+    default: cloudify-rest-service-3.4.0-m5_b394.x86_64.rpm
```

Comment:
for the reviewer. the names of the changed inputs have "url" in them even though the default values are just names of files. The reason for that is that if the user provides something different other than the current defaults, it must be a url. We could solve that by providing an additional set of inputs for file names and if a "url" input is provided, ignore the name input.. but that would both take time which we currently don't have an would load up the manager on additional unwanted inputs.